src/inmanta/db/versions/v202308010.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Copyright 2023 Inmanta
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    schema = """
+    -- Add the 'last_success' column
+    ALTER TABLE public.resource ADD COLUMN last_success TIMESTAMP WITH TIME ZONE;
+    """
+    await connection.execute(schema)
+
+    # For each environment
+    # For the latest released version
+    # Set the last success
+    update_query = """
+    UPDATE resource as r
+    SET last_success = (
+        SELECT max(finished)
```

Comment:
Related to other comment about using `finished` as marker. Though in this case it may be our best option regardless.

Reply:
modified